block-storage-attacher/tests/e2e/e2e-tests/utilscript.sh

```diff
@@ -23,8 +35,19 @@ elif [ "$2" = "voldelete" ]
 then
     export VOL_ID=$1
     export NOD_IP=$3
-    bx sl block access-revoke -p $NOD_IP $VOL_ID
-    bx sl  block volume-cancel -f --immediate $VOL_ID
+    slcli block access-revoke  $VOL_ID
+    slcli -y block volume-cancel  --immediate $VOL_ID
+elif [ "$2" = "portworxpvcreate" ]
+then
+    install_portworx_plugin
+    check_portworx_pod_state "portworx"
+    echo "BlockVolumeAttacher-Volume-Test: PortWorx Plugin-Installation: PASS" >> $E2E_PATH/e2eTests.txt
+    export CLSFILE=$1
+    kubectl  create -f $CLSFILE
+elif [ "$2" = "portworxdelete" ]
+then
+    curl -fsL https://install.portworx.com/px-wipe | bash
```

Comment:
Just before curl command do `kubectl delete sc <storage class>`. Delete class so when u create in next run it wont error out.